convert/circle/internal/orbs/slack/slack.go

```diff
@@ -34,13 +34,50 @@ func Convert(command string, step *circle.Custom) *harness.Step {
 // helper function converts a slack/notify
 // orb to a run step.
 func convertNotify(step *circle.Custom) *harness.Step {
+	customMessage, _ := step.Params["custom"].(string)
+	channel, _ := step.Params["channel"].(string)
+	if channel == "" {
+		channel = "$SLACK_DEFAULT_CHANNEL"
```

Comment:
how does `$SLACK_DEFAULT_CHANNEL` work?

Reply:
It's the env variable circle can set. My expectation was we'd handle it in the harness-migrate OR expect the SE to populate this value?

Reply:
take this feedback with a grain of salt, since I don't fully understand how circle works.  But if we want the plugin to pick up the value of `$SLACK_DEFAULT_CHANNEL` we can leave channel as an empty string and configure the plugin to read this variable:

```diff
		cli.StringFlag{
			Name:   "channel",
			Usage:  "slack channel",
-			EnvVar: "PLUGIN_CHANNEL",
+			EnvVar: "PLUGIN_CHANNEL,SLACK_DEFAULT_CHANNEL",
		},
```

If we set `channel = "$SLACK_DEFAULT_CHANNEL"` it will result in `PLUGIN_CHANNEL='$SLACK_DEFAULT_CHANNEL'"`

Reply:
if we look at the yaml provided by the company we wish to migrate, they are setting channel in the yaml, not via the env variable. So maybe it's best we just remove this and add it as an option in the slack plugin?